### ISSUE TITLE
xessD3D12BuildPipelines() returns XESS_RESULT_ERROR_NOT_IMPLEMENTED on Arc A380

### ISSUE THREAD

**Issue by DarioSamo** [intel/xess#4]
I believe Kaplanyan has already addressed this on Twitter, but it's probably worth repeating it here to be notified when the problem is solved and for other developers who might run into the issue and search about it.

The following call returned the correct result in my system with an NVIDIA GeForce RTX 3090, but a user with an A380 reported that `XESS_RESULT_ERROR_NOT_IMPLEMENTED` was returned instead as indicated on my process' log. It's worth mentioning that XeSS ran correctly for this user upon deleting the `xessD3D12BuildPipelines` call and letting `xessD3D12Init` do the work instead, at the expense of a longer startup time. Their drivers were also confirmed to be up to date and the XeSS demos ran as intended.

```
xessD3D12BuildPipelines(xessContext, nullptr, false, 
  XESS_INIT_FLAG_RESPONSIVE_PIXEL_MASK | XESS_INIT_FLAG_LDR_INPUT_COLOR);
```

I ran into this because the manual itself mentions the possibility of using the pipeline caching:

![image](https://user-images.githubusercontent.com/538504/196813291-4cd9da73-ecad-461d-b8b1-2eaa01062c0b.png)

But upon reading the included samples, `XeSSRuntime::InitializePipeline` is the only function to call `xessD3D12BuildPipelines`, but it's unused because `InitializePipeline` is never called by the demo application. `BasicSample` does not currently make use of the function either. Judging by its omission from the code in both samples, I figure the team is already aware of the problem.

Thanks for opening up the issue tracker!

**Comment by sunzhuoshi** [comment]
Issue confirmed, let me try if I can speed it up, thanks for reporting here.

**Comment by hsiversson** [comment]
Seeing this issue on v1.01 release as well using an Arc A770 on drivers 31.0.101.3490 & 31.0.101.3802.
Any updates on the progress @sunzhuoshi?

**Comment by sunzhuoshi** [comment]
Already fixed, should be available in v1.1.0.

**Comment by sunzhuoshi** [comment]
Version 1.1.0 just released: https://github.com/intel/xess/releases/tag/v1.1.0 @hsiversson
